science/rtl_433/Portfile

```diff
@@ -4,10 +4,7 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 
-github.setup        merbanan rtl_433 18.05
-version             18.05
-epoch               1
-
+name                rtl_433
```

Comment:
The `name` line is redundant because it will be set to that by the `github.setup` line. 

Honestly, I don't like that you moved it to just above the checksums, but if it works and the maintainer agrees, that is fine.

Reply:
I followed the other discussion; so the position is maintainer related.
Anyway, also version is redundant (set by `github.setup` too).

Reply:
Yes exactly!

Reply:
with this reasoning, `revision 0` is useless seen that is set by default.

Reply:
`revision 0` is useless! But some people like to have it there apparently. I personally don't agree, but I just do what everyone agrees to.

Reply:
ok ;-) I really like the principle "Conventions over Configurations" or "sane defaults"